### DISCUSSION TITLE
Axios library error

### DISCUSSION THREAD

**Discussion by arunVeeranna** [callstack/repack#502] [General]
When add axios i am getting below error  when we add repack and run through webpack. any suggestion
![Simulator Screenshot - iPhone 15 - 2024-02-13 at 12 28 48](https://github.com/callstack/repack/assets/124761100/e1de42e0-1c3d-4a1e-829f-d789bcfb29f5)

**Comment by jbroma** [comment]
hey @arunVeeranna, take a look here: https://github.com/callstack/repack/issues/359
